CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+## Set up
```

Comment:
We might want to context our thinking behind this - it's meant to demonstrate integrations. Integrations can be written in any language or framework, and we don't officially support this beyond that it's an example of how an add-on might be built.

Gauger can probably help with this language.

Reply:
this language probably belongs in the README, though. oops!